openfec/entities/annotate.py

```diff
@@ -1,132 +0,0 @@
-"""
-Tools for annotating records
```

Comment:
In the future, I'd like to see PR's kept to just what the original task is about, but since this stuff ought to be taken out anyway, and we're short on time, just leave it as-is.